boards/arm/zkit_arm_1769/doc/zkit-arm-1769.rst

```diff
@@ -0,0 +1,147 @@
+.. _zkit-arm-1769:
+
+ZKit-ARM-1769, ARM Development Board
+#####################################
+
+Overview
+********
+
+The ZKit-ARM-1769, ARM Development Kit features an ARM Cortex-M3 based LPC1769 MCU
+with a wide range of connectivity support and configurations. Here are
+some highlights of the ZKit-ARM-1769 board:
+
+- NXP LPC1769 micro-controller with 512KB Flash and 64KB RAM
+- 120MHz ARM CPU
+- Networking Interfaces:
+
+       - 10/100Mbps Ethernet Interface
+       - Two Channel CAN Bus Interface (2.0 B)
+       - RS-485 Bus Interface
+       - UART Interfaces (TTL and USB-Serial)
+- Device Interface:
+
+       - USB 2.0 Device Interface
+       - SPI Interface
+       - I2C Interface
+- Storage:
+
+       - microSD Connector
+       - 512KB on-chip Flash
+       - 2K I2C EEPROM
+- Analog Interface:
+
+       - ADC, 12-bit, 4 channels
+       - DAC, 10-bit, 1 channel
+       - PWM, 5 channels
+- User Interface:
+
+       - 128x64 graphics LCD, with backlight
+       - Five button keypad
+       - 2 debug LEDs
+- Flashing/Debugging Interface:
+
+       - UART serial console
+       - JTAG
+
+.. image:: img/zkit-arm-1769-top.jpg
+     :width: 640px
+     :align: center
+     :height: 500px
+     :alt: ZKit-ARM-1769
+
+More information about the board can be found at the `Zkit-ARM-1769 website`_.
+
+Supported Features
+==================
+
+The Zephyr ZKit-ARM-1769 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | clock                               |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
```

Comment:
not yet supported on Zephyr porting. -> not yet supported on Zephyr.

Reply:
changed